src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/repository/PrisonerBRepository.kt

```diff
@@ -0,0 +1,19 @@
+package uk.gov.justice.digital.hmpps.prisonersearch.repository
+
+import org.springframework.data.domain.Page
+import org.springframework.data.domain.Pageable
+import org.springframework.data.elasticsearch.annotations.Query
+import org.springframework.data.elasticsearch.repository.ElasticsearchRepository
+import uk.gov.justice.digital.hmpps.prisonersearch.model.Prisoner
+import uk.gov.justice.digital.hmpps.prisonersearch.model.PrisonerB
+import java.time.LocalDate
+
```

Comment:
For now having to double up the repo and class to provide the out of box spring data support - May be a better way!

Reply:
Yes @andymarke I'll have a go at that in next commit